internal/api/v1/applications.go

```diff
@@ -135,22 +134,13 @@ func (hc ApplicationsController) Update(w http.ResponseWriter, r *http.Request)
 		return APIErrors{BadRequest(err)}
 	}
 
-	if updateRequest.Instances == "" {
-		return APIErrors{NewAPIError("Instances not specified", "", http.StatusBadRequest)}
-	}
-
-	instances, err := strconv.Atoi(updateRequest.Instances)
-	if err != nil {
-		return APIErrors{BadRequest(err, "")}
-	}
-
-	if instances < 0 {
+	if updateRequest.Instances < 0 {
 		return APIErrors{NewAPIError(
```

Comment:
Could this use the `BadRequest` utility function ? (See sibling file `errors.go`).